addon/-private/drop-zone-state.js

```diff
@@ -44,11 +44,11 @@ export default EmberObject.extend({
 
   overlayText: computed('type', function() {
     let type = this.get('type');
-    return get(dropZone, `overlayText${ capitalize(type) }`);
+    return get(dropZone, `overlayText${capitalize(type)}`);
```

Comment:
These spaces disappearing is making me sad 😞

Reply:
We decided to live with this since there isn't much to do to get around spacing issues when using prettier & eslint together.